openpype/hosts/blender/api/capture.py

```diff
@@ -123,6 +131,51 @@ def capture(
     },
 }
 
+FILE_EXTENSIONS = {
+    "JPEG": ".jpg",
+    "JPEG2000": ".jpg",
+    "TARGA": ".tga",
+    "TARGA_RAW": ".tga",
+    "OPEN_EXR": ".exr",
+    "OPEN_EXR_MULTILAYER": ".exr",
+    "AVI_JPEG": ".avi",
+    "AVI_RAW": ".avi",
+    "FFMcPEG": {
+        "QUICKTIME": ".mov",
+        "MPEG4": ".mp4",
+        "MPEG2": ".mpg",
+        "MPEG1": ".mpg",
+        "FLASH": ".flv",
+    },
+}
+
+def get_extension_from_image_settings(image_settings):
```

Comment:
```suggestion
def get_extension_from_image_settings(image_settings:dict)->str:
```

Reply:
Arf okay, I have no typing here, I will update the entire code, docstrings, and comments.